content/docs/get-started-with-neon/connection-pooling.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Connection pooling
+---
+
+## Postgres connection limits
+
+Each postgres connection creates a new process in the operating system, which occuppies some amount of resources -- memory, open file descriptors, etc. Hence postgres has limits on amount of open connections. I in our free tier we allow 20 simultaneous postgres connections.
+
+Some apps tends to open a lot more connections and keep most of them inactive. This usually happens either because of database driver limitation or due to the lots of deployed instances of the app. To deal with such situatiations we use connection pooler.
```

Comment:
```suggestion
Some apps tend to open a lot more connections and keep most of them inactive. This usually happens either because of database driver limitation or due to the lots of deployed instances of the app. To deal with such situatiations, we use connection pooler.
```